### ISSUE TITLE
Fix docker image check script for images outside the usual set

### ISSUE THREAD

**Issue by joestringer** [cilium/cilium#15901]
When I run `make -C install/kubernetes check-docker-images` in the v1.10 branch, I see:

```
$ make -C install/kubernetes/ check-docker-images
docker.io/envoyproxy/envoy:v1.18.2@sha256:e8b37c1d75787dd1e712ff389b0d37337dc8a174a63bed9c34ba73359dc67da7 does not exist!
docker.io/cilium/hubble-ui:v0.7.7@sha256:f20b6b3074a431ecd2730cf19afad2562664cf98baecf32b4cbe2a09f04ad8fa does not exist!
quay.io/cilium/hubble-ui:v0.7.7@sha256:f20b6b3074a431ecd2730cf19afad2562664cf98baecf32b4cbe2a09f04ad8fa does not exist!
docker.io/cilium/hubble-ui-backend:v0.7.7@sha256:11d06121c86549c0c8684571ffed4ff3d4c4417f551728f4fad6d05525c47e9d does not exist!
quay.io/cilium/hubble-ui-backend:v0.7.7@sha256:11d06121c86549c0c8684571ffed4ff3d4c4417f551728f4fad6d05525c47e9d does not exist!
make: *** [Makefile:106: check-docker-images] Error 1
```

However, several of these images exist (I tried the envoy one and the quay versions of the hubble ones). We should fix up the script to properly locate these images.

**Comment by joestringer** [comment]
These might just be breaking due to a poor assumption around how you can split image names based on the `:` character position, which is a common way to break when the sha256 digests are specified.

**Comment by nickolaev** [comment]
This issue is no longer valid. I believe it might be fixed by 3ec10e7081724f76bf577c40e6f3aa62079dd8d2
Shall we close it @joestringer

**Comment by joestringer** [comment]
Yep, thanks for confirming.
